---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Exploiting CRISPR-Cas9 technology to investigate individual histone modifications
subtitle: ''
summary: ''
authors:
- Juan-Jos√© Vasquez
- Carolin Wedel
- Raul O. Cosentino
- T. Nicolai Siegel
tags:
- '"Cell Line"'
- '"CRISPR-Cas Systems"'
- '"Gene Editing"'
- '"Genome"'
- '"Protozoan"'
- '"Green Fluorescent Proteins"'
- '"Histone Code"'
- '"Histones"'
- '"Mutagenesis"'
- '"Site-Directed"'
- '"Organisms"'
- '"Genetically Modified"'
- '"Plasmids"'
- '"Protein Processing"'
- '"Post-Translational"'
- '"RNA"'
- '"Guide"'
- '"Trypanosoma brucei brucei"'
categories: []
date: '2018-10-01'
lastmod: 2021-06-03T18:30:40+02:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-06-03T16:30:39.939916Z'
publication_types:
- '2'
abstract: Despite their importance for most DNA-templated processes, the function
  of individual histone modifications has remained largely unknown because in vivo
  mutational analyses are lacking. The reason for this is that histone genes are encoded
  by multigene families and that tools to simultaneously edit multiple genomic loci
  with high efficiency are only now becoming available. To overcome these challenges,
  we have taken advantage of the power of CRISPR-Cas9 for precise genome editing and
  of the fact that most DNA repair in the protozoan parasite Trypanosoma brucei occurs
  via homologous recombination. By establishing an episome-based CRISPR-Cas9 system
  for T. brucei, we have edited wild type cells without inserting selectable markers,
  inserted a GFP tag between an ORF and its 3'UTR, deleted both alleles of a gene
  in a single transfection, and performed precise editing of genes that exist in multicopy
  arrays, replacing histone H4K4 with H4R4 in the absence of detectable off-target
  effects. The newly established genome editing toolbox allows for the generation
  of precise mutants without needing to change other regions of the genome, opening
  up opportunities to study the role of individual histone modifications, catalytic
  sites of enzymes or the regulatory potential of UTRs in their endogenous environments.
publication: '*Nucleic Acids Research*'
doi: 10.1093/nar/gky517
---
